packages/analytics/analytics-chart/src/components/chart-plugins/ChartLegend.vue

```diff
@@ -133,21 +136,26 @@ const position = inject('legendPosition', ref(ChartLegendPosition.Right))
   }
 
   &.horizontal {
-    column-gap: $kui-space-50;
-    flex-direction: row;
-    flex-wrap: wrap;
+    column-gap: $kui-space-30;
+    display: grid;
+    grid-template-columns: repeat(auto-fit, $kui-space-150);
     justify-content: center;
     margin-top: $kui-space-50;
-    max-height: 60px;
+    max-height: $kui-space-150;
+    row-gap: $kui-space-30;
```

Comment:
I would go aggressively small here, 4px for both `column-gap` and `row-gap`, which equates to`$kui-space-20`.

And on L187 , for `.legend-container li`, the margin all around could be as small as 8px (`$kui-space-40`)

<img width="961" alt="Screenshot 2023-09-11 at 5 06 29 PM" src="https://github.com/Kong/public-ui-components/assets/103061463/b57ceca9-2008-4160-9aad-e95a7176587f">